### PR TITLE
EIP1-3229: EIP1:4618: api spec for Get AED summaries

### PR DESCRIPTION
**Pull request by vishalgupta-valtech**
image

image

### CODE REVIEW

File: src/main/resources/openapi/PrintAPIs.yaml [hunk 1]
```diff
@@ -1049,32 +1050,71 @@ components:
...
+      required:
+        - addressee
+        - registeredAddress
+
     AnonymousElectorDocumentStatus:
       title: AnonymousElectorDocumentStatus
       description: Status of an Anonymous Elector document
       type: string
       enum:
-        - generated
```
Comment (vishalgupta-valtech) [inline src/main/resources/openapi/PrintAPIs.yaml:1077]: We never set this value anymore, so removed. All db records were already set to PRINTED in previous PRs

Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml:1077]: Good point, but I am not sure if they will ever get this back from api , I will ask @gavinthomas-valtech

**Review by sentioservices — APPROVED** [review]
LGTM, but if it is a breaking change for the FE then obviously worth letting them know

File: src/main/resources/openapi/PrintAPIs.yaml (hunk 1 above)
Comment (vishalgupta-valtech) [reply src/main/resources/openapi/PrintAPIs.yaml]: UI team is away therefore I have reverted change that removes `GENERATED` from api spec and entity

**Pull request merged**